PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Utils/BasicEmbedTokenGenerator.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  BasicEmbedTokenGenerator.swift
+//  ControlsLockScreenSampleApp
+//
+//  Created by Carlos Ceja on 10/18/17.
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class BasicEmbedTokenGenerator: NSObject, OOEmbedTokenGenerator {
+  
+  private var pcode: String
+  
+  private var apiKey: String
+  
+  private var apiSecret: String
+  
+  private var accountId: String
+  
+  private var authorizeHost: String
+  
+  init(pcode: String, apiKey: String, apiSecret: String, accountId: String, authorizeHost: String) {
+    self.pcode = pcode
+    self.apiKey = apiKey
+    self.apiSecret = apiSecret
+    self.accountId = accountId
+    self.authorizeHost = authorizeHost
+    super.init()
+  }
+  
+  func getApiKey() -> String {
```

Comment:
Same comments here as `PlayerSelectionOption`. No need to define getters.

Reply:
Done. Using public private(set) var FIELD.